storage.c

```diff
@@ -220,7 +220,7 @@ static struct pv_storage *pv_storage_new()
 	struct statfs buf;
 	struct pv_storage *this;
 
-	pv_paths_storage_config_file(path, PATH_MAX, PANTAHUB_FNAME);
+	pv_paths_storage_file(path, PATH_MAX, PVMOUNTED_FNAME);
```

Comment:
why this change?

Reply:
well, I imagined if we are going to use .pvmounted for disks too why not storing that in /storage instead of /storage/config